src/index.css

```
@@ -114,23 +114,24 @@
-    @apply mb-4 font-display text-lg font-bold tracking-wider text-base-content;
+    @apply mb-4 font-sans text-lg font-bold tracking-wider text-base-content;
   }
 
   .section-header-row {
     @apply mb-3 flex items-center justify-between;
   }
 
   .cta-button {
-    @apply btn btn-accent btn-lg w-full font-display text-base font-bold tracking-widest transition-all duration-200 hover:bg-base-content hover:text-base-100 active:scale-[0.98];
+    @apply btn btn-accent btn-lg w-full font-sans text-base font-bold tracking-widest transition-all duration-200 hover:bg-base-content hover:text-base-100 active:scale-[0.98];
+    clip-path: var(--clip-cut-corner);
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)